Sources/TensorFlow/Operators/NN.swift

```diff
@@ -384,15 +384,14 @@ func _vjpAvgPool3D<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
-/// Computes a 2-D convolution using `self` as input, with the specified
-/// filter, strides, and padding.
+/// Computes a 2-D convolution with the specified input, filter, strides, and padding.
 ///
 /// - Parameters:
 ///   - x: The input.
 ///   - filter: The convolution filter.
 ///   - strides: The strides of the sliding filter for each dimension of the input.
 ///   - padding: The padding for the operation.
-/// - Precondition: `self` must have rank 4.
+/// - Precondition: `input` must have rank `4`.
```

Comment:
Oh, there is no parameter called `input`. It's currently called `x`. Could you please help change the first parameter name to `input` as well as the doc comment on line 390?